src/common/popup/components/about.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import About from './about';
+
+describe('about', () => {
+  function render(/* overrides */) {
+    return shallow(<About />);
+  }
+
+  it('renders', () => {
+    expect(render({})).toMatchSnapshot();
```

Comment:
Have you thought about using inline snapshots (`toMatchInlineSnapshot`)? I see that the snapshots are a bit longer, but that would add some more information into the test file.

Reply:
🤔 I have not. I will give it a shot and see what that looks like 👍 Great suggestion.

Reply:
Mmh, looks a bit odd because the whole snapshot is not indented properly relative to the surrounding code… and it's really quite long. Maybe keeping it in a separate file is the easier option? What do you think @lwassermann?

PS: Since inline snapshots require prettier though and I've just set it up for trying it out, I'll open a new PR for the prettier integration.